config/projects/supermarket.rb

```diff
@@ -32,7 +32,7 @@
 override :bundler, version: "1.7.3"
 override :'chef-gem', version: "11.16.4"
 override :postgresql, version: "9.3.4"
-override :'omnibus-ctl', version: "nls/ignore-config"
+override :'omnibus-ctl', version: "master"
```

Comment:
Cool, I'll do that after it's merged.  As that branch doesn't exist anymore I had to yank the override to get the omnibus to build the package.